### PR TITLE
Document ChefVault Secret Management in Chef Automate

### PR DESCRIPTION
**Pull request by afiune**

### CODE REVIEW

File: chef_master/source/delivery_manage_secrets.rst [hunk 1]
```diff
@@ -14,12 +14,20 @@ Manage Secrets in a build-cookbook
 This topic describes how and why to use secrets in a ``build-cookbook``:
 
 * This code is used in the ``build-cookbook``
-* Secrets are managed at a project level
-* All of the secrets for a given project will live in the same encrypted data bag item
+* Secrets are managed at a project, organization and/or enterprise level
+* There are two mechanism to manage secrets in Chef Automate:
```
Comment (scottopherson) [inline chef_master/source/delivery_manage_secrets.rst]: mechanism -> mechanisms

File: chef_master/source/delivery_manage_secrets.rst [hunk 2]
```diff
@@ -14,12 +14,20 @@ Manage Secrets in a build-cookbook
...
+  - Using chef vaults 
 
 For more information on security, see `How to be a Secure Chef <https://learn.chef.io/skills/be-a-secure-chef/>`_.
 
-Create a Data Bag
+
+Using encrypted data bag
+=====================================================
+
+This section describes how to use encrypted data bag items in Chef Automate
```
Comment (scottopherson) [inline chef_master/source/delivery_manage_secrets.rst]: Needs a period.

File: chef_master/source/delivery_manage_secrets.rst [hunk 3]
```diff
@@ -33,9 +41,17 @@ If the project is in the ``chef`` enterprise and is in the ``CIA`` organization
 
    chef-CIA-chef-web-www
 
-This should use the same encrypted data bag item that is distributed with the build nodes.
+This should use the same encrypted data bag secret key that is distributed with the build nodes.
+
+If the project item does not exist, ``delivery-sugar`` will try to load the secretsi from the organization that your project lives in. It will look for an item called:
```
Comment (scottopherson) [inline chef_master/source/delivery_manage_secrets.rst]: secretsi -> secrets

File: chef_master/source/delivery_manage_secrets.rst [hunk 4]
```diff
@@ -65,3 +81,44 @@ From there, begin using the secrets by calling the ``get_project_secrets`` metho
   end
 
 which is part of the ``publish.rb`` recipe in the ``delivery-truck`` cookbook: https://github.com/chef-cookbooks/delivery-truck/blob/master/recipes/publish.rb#L91-L103.
+
+Using a Chef Vault
+=====================================================
+
+This section describes how to use Chef Vault in Chef Automate
```
Comment (scottopherson) [inline chef_master/source/delivery_manage_secrets.rst]: Needs a period.

**Comment by afiune** [comment]
@scottopherson Thanks!!! Fixed them all.. 🎉

**Review by scottopherson — APPROVED** [review]

**Comment by davidwrede** [comment]
gif-keyboard-4284435597990758335

File: chef_master/source/delivery_manage_secrets.rst [hunk 5]
```diff
@@ -64,4 +80,44 @@ From there, begin using the secrets by calling the ``get_project_secrets`` metho
...
+
+   chef
+   chef-cookbook
+   chef-cookbook-mysql
+
+During the creation of a Chef vault, the data in these vaults are merged into a single Ruby hash. Any duplicate key names will be merged as follows:
+   - ``<ENT>-<ORG>-<PROJECT>`` will overwrite ``<ENT>-<ORG>`` and ``<ENT>``.
+   - ``<ENT>-<ORG>`` will overwrite ``<ENT>``.
+
+Using an Encrypted Data Bag
```
Comment (afiune) [inline chef_master/source/delivery_manage_secrets.rst:112]: This is not an encrypted data bag. This is a Chef Vault... 🤔 We have to fix this.

**Comment by afiune** [comment]
@kagarmoe @davidwrede A little comment above ^^

**Pull request merged**